File: fix_tags.js
const fs = require("fs");
const path = require("path");
const root = path.join(__dirname, "frontend", "src");
const endTag = "</" + "div>";

function walk(dir) {
  for (const name of fs.readdirSync(dir)) {
    const full = path.join(dir, name);
    if (fs.statSync(full).isDirectory()) walk(full);
    else if (name.endsWith(".tsx")) {
      let t = fs.readFileSync(full, "utf8");
      if (t.includes("motion")) {
        t = t.replace(/<\/motion>/g, endTag);
        fs.writeFileSync(full, t);
        console.log("fixed", full);
      }
    }
  }
}
walk(root);

File: fix_tags.py
from pathlib import Path

root = Path(r"c:\Users\arche\Desktop\ATOMBERG\atomquest\frontend\src")
close_div = "</" + "div>"
for path in root.rglob("*.tsx"):
    text = path.read_text(encoding="utf-8")
    if "motion" in text:
        text = text.replace("</motion>", close_div)
        path.write_text(text, encoding="utf-8")
        print("fixed", path)
